db/migrate/20131210202928_update_log_collection_path_in_configurations_settings.rb

```diff
@@ -4,8 +4,8 @@ class Configuration < ActiveRecord::Base
     self.inheritance_column = :_type_disabled # disable STI
   end
 
-  NEW_PG_CONF_PATH = "/opt/rh/postgresql92/root/var/lib/pgsql/data/*.conf"
-  NEW_PG_LOGS_PATH = "/opt/rh/postgresql92/root/var/lib/pgsql/data/pg_log/*"
+  NEW_PG_CONF_PATH = "/opt/rh/rh-postgresql94/root/var/lib/pgsql/data/*.conf"
+  NEW_PG_LOGS_PATH = "/opt/rh/rh-postgresql94/root/var/lib/pgsql/data/pg_log/*"
```

Comment:
Do we really want to change this historical migration?

.. and not make a similar change for currently-up-to-date DBs?